src/main/java/Echoa/Task.java

```diff
@@ -0,0 +1,62 @@
+package Echoa;
+
+/**
+ * Task is a class that encapsulates the characteristics of a Task.
+ * It contains the field, description and isDone.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone = false;
+
+    public Task(String description) {
+        this.description = description;
+    }
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    /**
+     * The method marks the task as done.
+     */
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * The method marks the task as not done.
+     */
+    public void unmarkAsUndone() {
+        this.isDone = false;
+    }
+
+    /**
+     * The method converts the isDone status to an icon,
+     * with X representing done.
+     *
+     * @return String representation of status icon.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
```

Comment:
Could consider changing this to `private` instead since it is only used within the same class.